pkg/broker/config/config.go

```diff
@@ -74,3 +85,13 @@ type Targets interface {
 func BrokerKey(namespace, name string) string {
 	return namespace + "/" + name
 }
+
+// TriggerKey returns the key of a trigger. Format is namespace/brokerName/targetName.
+func TriggerKey(namespace, broker, target string) string {
+	return fmt.Sprintf("%s/%s/%s", namespace, broker, target)
+}
+
+func SplitTriggerKey(key string) (string, string, string) {
```

Comment:
nit: comment
Also, could you check if `SplitBrokerKey` would be useful?

Reply:
Checked, it seems there is no need to add `SplitBrokerKey` now. TargetsConfig uses brokeKey as the key to store broker, so brokerName itself is not that needed. Targets are stored with key of target name in the map under a broker, so `SplitTriggerKey` would help to get the targetName for that.